### ISSUE TITLE
Optimum checkpoint

### ISSUE THREAD

**Issue by 6330A** [NLP2CT/TransGEC#1]
Hello! Your work is great! I ran into a problem when I tried to duplicate result. While fine-tuning with translationese data, I found that the model was saved according to the save_step setting, as per sh /shell_finetune-T5/train_zh.sh. This saves multiple checkpoints, but how do you determine which one is the best when generating and evaluating? Is the last saved model the best model? Or is there another way? Can you tell me? I was hoping you could help me? Thank you very much!

**Comment by tylefang** [comment]
Utilizing the validation set.
